yorkie/log/log.go

```diff
@@ -18,18 +18,53 @@ package log
 
 import (
 	"os"
+	"sync"
 
 	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
 )
 
-// Logger is the default logger used by Yorkie.
-var Logger *zap.SugaredLogger
+var defaultLogger *zap.SugaredLogger
+var coreLogger zapcore.Core
+var rawLogger *zap.Logger
 
-// Core is a minimal, fast logger interface.
-var Core zapcore.Core
+var logLevel = zapcore.InfoLevel
+var loggerOnce sync.Once
 
-var rawLogger *zap.Logger
+// DefaultLogger returns the default logger used by Yorkie.
+func DefaultLogger() *zap.SugaredLogger {
```

Comment:
DefaultLogger() -> Logger()
CoreLogger() -> Core()

What do you think of this change?

Reply:
I changed it because `log` package name already implied logger and more simplified :) thx!